services/CryptoUpdate.proto

```diff
@@ -53,4 +53,5 @@ message CryptoUpdateTransactionBody {
         google.protobuf.BoolValue receiverSigRequiredWrapper = 13; // If true, this account's key must sign any transaction depositing into this account (in addition to all withdrawals)
     }
     google.protobuf.StringValue memo = 14; // If set, the new memo to be associated with the account (UTF-8 encoding max 100 bytes)
+    google.protobuf.UInt32Value max_automatic_token_associations = 15; // The maximum number of tokens that an Account can be implicitly associated with. Up to a 1000 including implicit and explicit associations.
```

Comment:
I'm confused by the second part of this comment.  Is this saying the total max associations is 1000?  I would just have it similar to how it is in `CryptoCreate`